### DISCUSSION TITLE
Rust 程序库生态合作的例子 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#123] [Comments]
# Rust 程序库生态合作的例子 | 夜天之书

近期主要时间都在适应产品市场（Product Marketing）的新角色，不少想法还在酝酿和斟酌当中，于是文章输出没有太多时间来推敲和选题，只能保持每月发布相关的进展或一些零碎的思考。或许我可以恢复最早的模式，多做更新但是文章内容可能不会太过完整。 原本这一期想讨论的是 ASF 开源项目代码的所有权，以及开源软件变更协议的具体含义与操作方式。但是这个话题稍显枯燥，而且要把相关细节讲清楚，还需要继

https://www.tisonkun.org/2024/05/31/rust-ecosystem-and-collaboration/

**Comment by ZoomQuiet** [comment]
所以, 世界果然是个草台班子;
在开始构建这么多包含极端精巧以及意外的依赖问题网络之前,
构造新库的作者会不知道这些问题的潜在可能嘛?

也许吧,

但是, 在问题被发觉的第一时间, 大家有主动坐在一起商量解决嘛?

应该没有...

毕竟 Rust 定将吞噬世界, Rustaceans 又是最聪明的一大波好汉, 到时一定能自行解决的;

于是, 就这么任性的开始爆炸各种神奇的人造问题集了吧.

整体上来的, 就像是智能浪费, 大家故意相互制造有趣而无用的困难?

**Comment by Ray-Eldath** [comment]
结尾的两张图似乎挂了？

**Reply by tisonkun** [reply]
嗯，刚才发现之前没上传；补上了。
